lib/app/features/auth/views/pages/recover_user_twofa_page/components/twofa_options_selector_button.dart

```diff
@@ -12,13 +12,15 @@ class TwoFaOptionsSelectorButton extends StatelessWidget {
     required this.isOpened,
     required this.controller,
     required this.optionIndex,
+    this.enabled = true,
```

Comment:
Do we need to visually set it as `disabled` in case it's disabled?

Reply:
discussed